### PR TITLE
ci: perfeact mirror action

### LINKED ISSUE #29120: new branch trigger gitee
每次新建分支都触发 Gitee，可以优化一下~

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/29120

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |    -       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/mirror.yml [hunk 1]
```diff
@@ -1,12 +1,29 @@
 name: 🔀 Mirror to Gitee
 
-on: [ push, delete, create ]
+on:
```
Comment (xrkffgg) [inline .github/workflows/mirror.yml:3]: 这几个分支还有 tag 应该都不会进行删除操作，所以就移除了 delete

File: .github/workflows/mirror.yml [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+  push:
+    branches:
```
Comment (xrkffgg) [inline .github/workflows/mirror.yml:5]:
```suggestion
    branches:
```

File: .github/workflows/mirror.yml (hunk 1 above)
Comment (kerm1it) [reply .github/workflows/mirror.yml:3]: 创建分支会触发下面的 create 么？

File: .github/workflows/mirror.yml [hunk 3]
```diff
@@ -1,12 +1,29 @@
...
     steps:
+      - if: github.event_name == 'push'
+      - uses: wearerequired/git-mirror-action@v1
+        env:
+          SSH_PRIVATE_KEY: ${{ secrets.GITEE_SSH_PRIVATE_KEY }}
+        with:
+          source-repo: 'git@github.com:ant-design/ant-design.git'
+          destination-repo: 'git@gitee.com:ant-design/ant-design.git'
+
+      - if: github.event_name == 'create' && github.event.ref_type == 'tag'
```
Comment (kerm1it) [inline .github/workflows/mirror.yml:26]: 你这是把两个或条件分开了是吧？

Comment (xrkffgg) [reply .github/workflows/mirror.yml:26]: 是啊，上面的是限制 push 的。

下面的是 新建分支或者新建 tag 都会触发，然后限定为 tag

**Pull request merged**